conf/core-site.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
```

Comment:
Does this property need to be added for Hue to work? If not, we should remove it for the reasons below.

I believe fs.default.name can be placed in hdfs-site.xml (the configuration can be ported there per @brndnmtthws) although we need to test it . This does need to be configurable based on the configured cluster name. Also, we need to update the readme to let the user know that hadoop fs -ls / will work as opposed to hadoop fs -ls hdfs://hdfs/.